docs/docs/repository-settings.md

```diff
@@ -0,0 +1,50 @@
+# Repository settings
+
+All these settings are applied to this repository.
+
+| Setting | Value | Info |
+|-|-|-|
+| Allow merge commits | `enabled` | Default to pull request title and description |
+| Allow squash merging | `disabled` | |
+| Allow rebase merging | `disabled` | |
+| Always suggest updating pull request branches | `enabled` | |
+| Allow auto-merge | `enabled` | |
+| Automatically delete head branches | `enabled` | |
+| Projects | `disabled` | |
+| Issues | `disabled` | |
+| Allow | `enabled` | |
+
+## Add collaborators
+
+Default GitHub teams will be added as collaborator in the repositories granting access to all project members to contribute to the onboarded repositories.
+
+| Team | Role |
+|-|-|
+| [oauth-proxy-contributors](https://github.com/orgs/philips-forks/teams/oauth-proxy-contributors) | Maintain |
+
+## Create protection rules
+
+These are the protection rules for the `main` and `edi-foundation-integration` branches in order to secury and standarize the way of working.
```

Comment:
it's `master` branch here and not `main`

Reply:
Indeed, renamed to master.